tests/unit/decoder/src/test_decode_manifest.c

```diff
@@ -365,15 +365,93 @@ static uint8_t cbor_envelope_with_6_components[] = {
 			0x40, /* bytes(0) */
 
 	0x03, /* suit-manifest */
-	0x58, 0x1c, /* bytes(28) */
+	0x58, 0x2f, /* bytes(37) */
 	0xa3, /* map (3 elements) */
 	0x01, /* suit-manifest-version */ 0x01,
 	0x02, /* suit-manifest-sequence-number */ 0x10,
 	0x03, /* suit-common */
-		0x55, /* bytes(21) */
+		0x58, 0x27, /* bytes(30) */
```

Comment:
The length field is rather incorrect.
`13 * 3 + 3 = 42 (0x2a)` ?

Reply:
The comment was wrong. Updated the value.